vuepress/docs/next/tutorials/samples/install-standard-demo.md

```diff
@@ -8,18 +8,29 @@ sidebarDepth: 2
 
 ## Introduction
 
-This tutorial will guide you through installing a demo application using the Entando Component Repository and a set of Entando
-bundles. This solution template includes microservices, micro frontends, multiple pages, and CMS content. The goal of this exercise
-is to demonstrate how Entando bundles can be used to quickly install and create functionality in an Entando Application, enable portable business capabilities, and allow developers to reuse full stack operations via bundles.
+This tutorial will guide you through installing a demo application using the Entando Component Repository (ECR) and a set of Entando
+bundles. This solution template includes 
 
-Some of the key elements of the template are reviewed in
-the [application details section](#application-details) below.
+- microservices
+- micro frontends
+- multiple pages
+- CMS content
+
+The goal of this exercise is to demonstrate how Entando bundles can be used to 
+
+- quickly install and create functionality in an Entando Application
+- enable packaged business capabilities
+- allow developers to reuse full stack operations via bundles
+
+Some of the key elements of the template are reviewed in the [application details section](#application-details) below.
```

Comment:
I think Application Details should be capitalized as the name of the section below. I think it reads that better and matches the left nav that way.